Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -1515,6 +1539,15 @@ def com_google_fonts_check_name_license(ttFont, license_filename):
                 value = "https://".join(value.split("http://"))
                 http_warn = True
 
+            if "scripts.sil.org/OFL" in value:
+                passed = True
+                yield WARN, Message(
+                    "old-old-url",
```

Comment:
Typo?

Reply:
very-very-old-url !

LOL, you're right. That's a typo. I'll fix it. Thanks